### ISSUE TITLE
Autocomplete: Explore changing how results are announced to screen readers

### ISSUE THREAD

**Issue by davidakennedy** [CMSgov/design-system#936]
**Description**

In recent testing with a screen reader user with a few products on healthcare.gov that use the autocomplete, they got blocked. The results didn't announce all the time, presumably because of the `aria-live="polite"` setting. Also, when. they did, it said something like `2 results found` vs. something more specific like `2 zip codes found`.

Updating this could lead to a better experience because it's more specific to the results being displayed instead of being generic.

**Additional information [optional]**

It doesn't look like this would be an easy enhancement. The announcement comes from third-party code.

- The instructions come from an aria-live area on the page. This comes from [Downshift](https://github.com/downshift-js/]). Also, see [geta11ystatusmessage](https://github.com/downshift-js/downshift#geta11ystatusmessage). We should be able to alter that message to make it more specific, like Kristen mentions (2 zip codes found vs. 2 results found).
- It looks like we'd have to put a PR into Downshift, or find another way to change the message.

I wanted to explore the possibility of this, since it came from user testing.

Jira tickets `WNMGWINSHOP-761` and `FLH-334` have additional context.

**Comment by line47** [comment]
https://jira.cms.gov/browse/WNMGDS-924

**Comment by pwolfert** [comment]
Updated the ticket with more information

**Comment by forrestbaer** [comment]
@SparrowSpit would you mind weighing in on this when you have a chance?

**Comment by SparrowSpit** [comment]
I'm pretty sure this is related to [WNMGDS-2364](https://jira.cms.gov/browse/WNMGDS-2364) which is trying to address the oddities in what's being announced.

**Comment by pwolfert** [comment]
Good call, @SparrowSpit. Since we have [WNMGDS-924](https://jira.cms.gov/browse/WNMGDS-924) to track the issue of specific language, and we passed accessibility testing on [WNMGDS-2364](https://jira.cms.gov/browse/WNMGDS-2364), I'm going to go ahead and close this.
